packages/edit-site/src/components/template-details/index.js

```diff
@@ -101,23 +90,11 @@ export default function TemplateDetails( { template, onClose } ) {
 
 			<Button
 				className="edit-site-template-details__show-all-button"
-				{ ...( newMenuSidebar
-					? {
-							href: addQueryArgs( 'edit.php', {
-								// TODO: We should update this to filter by template part's areas as well.
-								post_type: template.type,
-							} ),
-					  }
-					: {
-							onClick: showTemplateInSidebar,
-							'aria-label': sprintf(
-								/* translators: %1$s: the template part's area name ("Headers", "Sidebars") or "templates". */
-								__(
-									'Browse all %1$s. This will open the %1$s menu in the navigation side panel.'
-								),
-								templateSubMenu.title
-							),
-					  } ) }
+				href={ addQueryArgs( '', {
+					page: 'gutenberg-edit-site',
```

Comment:
This JavaScript code will end up in Core where the site editor will be at `site-editor.php`, not `themes.php?page=gutenberg-edit-site`. This code needs to be updated to add `postType` to the current `location.href`.

Reply:
Fix here: https://github.com/WordPress/gutenberg/pull/36705